apis/core/lib/domain/observations/lib/index.ts

```diff
@@ -38,8 +38,19 @@ export function populateFilters(view: View, filters: AnyPointer): void {
     })
 }
 
-export function createView(cube: Cube): View {
-  return CubeQuery.View.fromCube(cube)
+export function createView(cube: Cube, pageSize: number): View {
+  const view = CubeQuery.View.fromCube(cube)
+
+  view.ptr.addOut(ns.view.projection, projection => {
+    const order = projection.blankNode()
+      .addOut(ns.view.dimension, view.dimensions[0].ptr)
+      .addOut(ns.view.direction, ns.view.Ascending)
```

Comment:
For now hardcodes ascending order over the first dimension (the observation ID I believe)